docs/source/collect/longhorn.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Longhorn
+description: Include information about Longhorn CSI
+---
+
+This collector will add information about the [longhorn](https://longhorn.io/) storage provider in a cluster.
+
+## Parameters
+
+The longhorn collector has the following parameters:
+
+#### `collectorName` (Optional)
+The name of the collector.
+
+#### `namespace` (Optional)
+The namespace where longhorn is running.
+If this is not provided, it will default to `longhorn-system`.
+
+## Example Collector Definition
+
+```yaml
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: sample
+spec:
+  collectors:
+    - longhorn: {}
+```
+
+
+## Included resources
+
+> Note: When an alternative namespace is specified, it will be used as the second element in the path of all resources. For example, settings would be found under `/longhorn/other-namespace/settings.yaml` instead of `/longhorn/settings.yaml`.
```

Comment:
Is it possible to include the namespace in the path regardless of whether it's `longhorn-system` or something else? Then it would be consistent, we do this [elsewhere](https://troubleshoot.sh/docs/collect/copy/#included-resources), and the paths in this doc would always be true, even if you didn't stop to read the note about the alternative namespace.

Reply:
Yes, it's possible. We'd have to update troubleshoot.

Reply:
Like a big update? It would be nice for it to be consistent so that we don't have to caveat it, but I don't know how big of an effort that is. Alternatively, we could put it in parentheses like we do [here](https://troubleshoot.sh/docs/collect/ceph/#included-resources) to indicate that it's optional.

Reply:
No, probably not too big.

Reply:
Ok, let's go for it unless you disagree.

Reply:
Updated